tabs/src/translations/en-US.json

```diff
@@ -56,7 +56,8 @@
             "TeamsTab.Delete.Chat": "Delete this chat's tabs.",
             "TeamsTab.ReadWrite.Chat": "Manage this chat's tabs.",
             "TeamsAppInstallation.Read.Chat": "Get a list of the apps that are installed in this chat.",
-            "OnlineMeeting.ReadBasic.Chat": "Get basic properties, such as name, schedule, organizer, and join link of a meeting associated with this chat."
+            "OnlineMeeting.ReadBasic.Chat": "Get basic properties, such as name, schedule, organizer, and join link of a meeting associated with this chat.",
+            "No resource available to grant permissions to" : "No resource available to grant permissions to."
```

Comment:
Can we change this to `"No resource (team or chat) available to grant permissions to"`? To me, it's not immediately obvious what a resource is.

Reply:
9a79adfaa8ba79e5d98756da5165b9167b8fac69